internal/uvm/network_test.go

```diff
@@ -0,0 +1,64 @@
+//go:build windows
+
+package uvm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/Microsoft/hcsshim/internal/hns"
+)
+
+func Test_SortEndpoints(t *testing.T) {
+	type config struct {
+		endpointNames []string
+		targetName    string
+	}
+	tests := []config{
```

Comment:
Could we also add a test case with empty endpoint array?

Reply:
Added!